tests/union_underlying_type_test.fbs

```diff
@@ -0,0 +1,17 @@
+
+namespace UnionUnderlyingType;
+
+table A {
+    a: int;
+}
+table B {
+    b: string;
+}
+table C {
+    c: bool;
+}
+union ABC: int { A = 555, B = 666, C = 777}
```

Comment:
So how do other languages handle this? Wouldn't they generate code to read a `uint8_t` type of the enum and have an error?